### PR TITLE
Some bugs in tests fixed

### PR DESCRIPTION
**Pull request by faucomte**
- Solution for level 73 is the correct one and player gets 20/20.
- Bug in level 104 where the Python editor would only read 25 lines has been fixed.
- Bug where tests wouldn't wait until the workspace had been fully loaded has been fixed - a message now pops up prompting the player to see the solution after it's effectively been loaded.
- Some selenium tests run faster.

### CODE REVIEW

**Comment by CelineBoudier** [comment]
@faucomte please rebase :)

File: game/end_to_end_tests/game_page.py [hunk 1]
```diff
@@ -86,7 +86,10 @@ def step(self):
 
     def solution_button(self):
         self.browser.find_element_by_id("solution_tab").click()
-        time.sleep(1)
+        popup_message = self.browser.find_element_by_id("myModal-title").text
+        while not popup_message == "View solution":
```
Comment (CelineBoudier) [inline game/end_to_end_tests/game_page.py]: this could potentially run forever... I'd add a timeout

Comment (CelineBoudier) [inline game/end_to_end_tests/game_page.py]: not sure if popups are good there as teachers will see it outside of tests too .... we could add a parameter but then the tests won't test the "actual" feature...

**Comment by faucomte** [comment]
**Tests runtime:**
Before: 165 tests run in 35 minutes (using Firefox Webdriver)
Now: 165 tests run in 18.5 minutes (using Chromedriver, basically half the time!)
Travis runs it all in 25.5 minutes (it uses Fireforx Webdriver too)
(see issue #847 for more info)

**Coverage:**
Decreased by 0.07% because of a break added for the while loop.
However most of the missing coverage is due to the test_random_road.py file which has not been edited in this pull request.

**Pull request merged**